DfE.FindInformationAcademiesTrusts/Pages/Trusts/Contacts/ContactsAreaModel.cs

```diff
@@ -5,30 +5,36 @@
 using DfE.FindInformationAcademiesTrusts.Services.Trust;
 using Microsoft.AspNetCore.Mvc;
 
-namespace DfE.FindInformationAcademiesTrusts.Pages.Trusts;
+namespace DfE.FindInformationAcademiesTrusts.Pages.Trusts.Contacts;
 
-public class ContactsModel(
+public class ContactAreaModel(
     IDataSourceService dataSourceService,
-    ILogger<ContactsModel> logger,
-    ITrustService trustService)
-    : TrustsAreaModel(dataSourceService, trustService, logger, "Contacts")
+    ITrustService trustService,
+    ILogger<ContactAreaModel> logger,
+    Func<TrustSubNavigationLinkModel, bool> setActivePage
+)
+    : TrustsAreaModel(dataSourceService, trustService, logger, ViewConstants.ContactsPageTitle)
 {
     public Person? ChairOfTrustees { get; set; }
     public Person? AccountingOfficer { get; set; }
     public Person? ChiefFinancialOfficer { get; set; }
     public InternalContact? SfsoLead { get; set; }
     public InternalContact? TrustRelationshipManager { get; set; }
 
-    public const string ContactNameNotAvailableMessage = "No contact name available";
-
-    public const string ContactEmailNotAvailableMessage = "No contact email available";
-
     public override async Task<IActionResult> OnGetAsync()
     {
         var pageResult = await base.OnGetAsync();
 
         if (pageResult.GetType() == typeof(NotFoundResult)) return pageResult;
 
+        SubNavigationLinks =
+        [
+            new TrustSubNavigationLinkModel("In DfE", "./InDfE", Uid, ViewConstants.ContactsPageTitle),
```

Comment:
We shouldn't need to extract the page title to `ViewConstants` as it's only ever used directly by this class.
We could either make it a constant of this class or inline it in the constructor and use `PageName` here
```suggestion
            new TrustSubNavigationLinkModel("In DfE", "./InDfE", Uid, PageName),
```